tools/scripts/seed/seedAuthUser.js

```diff
@@ -0,0 +1,101 @@
+const path = require('path');
```

Comment:
Not sure if this file even belongs in this dir? @raisedadead can you please look at the `package.json` file in this dir? It's pretty specific to seeding the challenges. Do you want me to change that? Or put this file in its own dir? I'm open to suggestions on how to organize this.

Reply:
Yes, you are correct. These things weren't cleaned up well. I propose we put all the seeding scripts in this same directory. Most likely the devDeps around this scripts are going to be more or less same. It makes sense to keep them together.

So, indeed the `package.json` in the directory needs cleanup. That's not a concern of this PR per se, yet I welcome to restructure/change it as you please. Thanks for taking the additional effort cleaning it up. If you get too confused on that, do not worry I can make the changes.

Essentially we would need to be able to reference this from the root.